src/Lykke.Service.HFT.AzureRepositories/LimitOrderStateRepository.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Async;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AzureStorage;
+using Lykke.Service.HFT.Core.Domain;
+using Lykke.Service.HFT.Core.Repositories;
+
+namespace Lykke.Service.HFT.AzureRepositories
+{
+    public class LimitOrderStateRepository : ILimitOrderStateRepository
+    {
+        private readonly INoSQLTableStorage<LimitOrderStateEntity> _orderStateTable;
+
+        public LimitOrderStateRepository(
+            INoSQLTableStorage<LimitOrderStateEntity> orderStateTable)
+        {
+            _orderStateTable = orderStateTable;
+        }
+
+        public async Task<ILimitOrderState> GetAsync(string clientId, Guid orderId)
+        {
+            return await _orderStateTable.GetDataAsync(clientId, orderId.ToString());
```

Comment:
Task can be returned without async/await

Reply:
Sure, but in this case it would not be possible because this method returns Task<ILimitOrderState>, but _orderStateTable.GetDataAsync returns Task<LimitOrderStateEntity>.